components/tools/OmeroWeb/test/integration/test_plategrid.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2015 Glencoe Software, Inc.
+# All rights reserved.
+#
+# Use is subject to license terms supplied in LICENSE.txt
+
+"""
+   Integration tests for the "plategrid" module.
+"""
+
+import pytest
+import test.integration.library as lib
```

Comment:
The library is not longer here, @sbesson relocated it.
`import library as lib`
should work now.